HighwayPatrol.cpp

```diff
@@ -28,10 +29,11 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     std::cout << name << ": vehicle is traveling " << v->speed - h->speedLimit << " miles per hour over the speed limit" << std::endl;
     if( willArrest )
     {
-        assert(false);
+        //assert(false);
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << v->name << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
use a `std::string` variable inside this `if` block.
use casting to figure out the TYPE of `v`.
then populate the string accordingly.
`v->name` is the name of the person driving the vehicle, not the vehicle type
- [ ] revise